master-libvirt/master.cfg

```diff
@@ -19,6 +19,7 @@ sys.setrecursionlimit(10000)
 sys.path.insert(0, "/srv/buildbot/master")
 
 from utils import *
```

Comment:
Should we take that refactor opportunity to remove the wildcard import?

Reply:
master.cfg will come next :)